### ISSUE TITLE
App keeps crashing in 1.0.10 after failed "reveal identity" in <1.0.10

### ISSUE THREAD

**Issue by slush0** [vexl-it/vexl#129]
How to reproduce:

1. Have a version prior 1.0.10
2. Ask for reveal identity for peer on version 1.0.10
3. Application keep crashing, even on app restart
4. Upgrade to 1.0.10
5. Application keep crashing, even on app restart

How it should behave:
We cannot fix compatibility in between <1.0.10 and 1.0.10+ in reveal identity feature, however, after updating, the application should recover from this issue and should start normally.

**Comment by leomira** [comment]
This is most probably not a problem exclusive to prior versions of the app.
I managed to get back to a bricked app (an instance that failed and kept failing after an identity reveal) in a chat offer1 vs. offer2. 

After I got back to an app I tried to reveal an identity in a chat in a regime offer2 vs. offer2. Now the requested phone failed. 

So it seems on Android this is a general issue: after the reveal identity the app that was requested keeps failing.
